app/static/admin/js/pages/analytics/analytics-main.js

```diff
@@ -1,36 +1,75 @@
+/*global moment*/
 "use strict";
 
 define(function(require) {
+  var quickSummary = require("./quick-summary");
+  var itemStatusPieChart = require("./item-status");
+  var locationBarChart = require("./item-location");
+  var itemNumberLineChart = require("./item-number");
+  var dateValueLineChart = require("./date-value");
+
   var quickSummaryDom = {
     ranged: {
-      donor: "#newDonors",
-      donation: "#newDonations",
-      item: "#newItems"
+      donor: "#rangedDonors",
+      donation: "#rangedDonations",
+      item: "#rangedItems",
+      value: "#rangedValues"
     },
     total: {
       donor: "#totalDonors",
       donation: "#totalDonations",
-      item: "#totalItems"
+      item: "#totalItems",
+      value: "#totalValues"
     },
     calculated: {
       donationPerDonor: "#avgDonationPerDonor",
-      itemPerDonation: "#avgItemPerDonation"
+      temPerDonor: "#avgItemPerDonor",
+      itemPerDonation: "#avgItemPerDonation",
+      valuePerItem: "#avgValuePerItem",
     }
   };
 
-  var quickSummary = require("./quick-summary");
-  var itemStatusPieChart = require("./item-status");
-  var locationBarChart = require("./item-location");
-  var itemNumberLineChart = require("./item-number");
+  // dateRangePicker initialization
+  var start = moment().subtract(2, 'years');
+  var end = moment().subtract(1, 'years');
```

Comment:
I think it would be best to default the start and end to be `moment().subtract(1,'year')` and `moment()`